app/models/namespace.rb

```diff
@@ -219,6 +219,27 @@ def validate_nesting_level
     errors.add(:parent_id, 'nesting level too deep')
   end
 
+  # self = namespace receiving transferred_namespace
+  # old_namespace = the old namespace transferred_namespace originated from
+  def update_metadata_summary_by_namespace_transfer(transferred_namespace, old_namespace)
+    metadata_to_update = transferred_namespace.metadata_summary
+    return if metadata_to_update.empty?
+
+    unless old_namespace.nil? || old_namespace.type == Namespaces::UserNamespace.sti_name
+      subtract_from_metadata_summary_count(old_namespace.self_and_ancestors, metadata_to_update, false)
+    end
+
+    return unless type != Namespaces::UserNamespace.sti_name
```

Comment:
We would still want to go through the block on line 228 if the `self`/receiving namespace is `User` because that block on 228 is checking the old namespaces that the transferred namespaces originated from.

So if a project is getting transferred from a group to a user namespace, we would want to go through 228 still and update the group/its ancestors by subtraction.

Reply:
Sorry about that I missed the `old_namespace` in `old_namespace.type == Namespaces::UserNamespace.sti_name`. Disregard the comment above